opus/application/apps/help/templates/help/splash.html

```diff
@@ -0,0 +1,15 @@
+<table>
+    <tr>
+        <td style="border:0;"><img src="{{ STATIC_URL }}img/splash-outer-planets.jpg" width=250></td>
```

Comment:
all styling should be css and live in a css file - might be prudent to create a new css file for just the splash.
Not entirely sure why you would use a table to format the data; probably divs a better option and would provide a lot more control.

Reply:
I'm going to leave this one for future cleanup by someone who knows what they're doing

Reply:
ok, just make an issue for it so it doesn't get lost